ref_cache/main.c

```diff
@@ -24,6 +24,12 @@ DEALINGS IN THE SOFTWARE.  */
 
 #include <config.h>
 
+#if defined(__APPLE__) && !defined(_DARWIN_C_SOURCE)
```

Comment:
I'd love it if you could guard this with ` && __MACOS_SDK_VERSION < whatever`, but in typical Apple fashion their SDKs don't appear to define any SDK version macros. Le big sigh…